validator/testdata/amp4ads_feature_tests/amp_selector.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```

Comment:
nit: update to 2019. don't need to run update-tests again, just update in the .out file too

Reply:
Done. Thanks!